src/ol/format/WFS.js

```diff
@@ -262,7 +321,7 @@ class WFS extends XMLFeature {
   readFeaturesFromNode(node, opt_options) {
     /** @type {import("../xml.js").NodeStackItem} */
     const context = {
-      node: node,
+      node,
```

Comment:
I didn’t confirm, but I’m assuming the shorthand gets transpiled away for IE.

Reply:
Also my conclusion but also not confirmed.